app/controllers/public/home_controller.rb

```diff
@@ -52,14 +52,13 @@ def create
 
       transaction = Payment.new(
         description: I18n.t("form.introduction", user: @member.name),
-        amount: total,
+        amount: total - Settings.mongoose_ideal_costs, # Transaction costs are paid by the association
```

Comment:
This would have been nice to have merged before the start of the summer 😓

Reply:
Uhu